polkadot/node/subsystem-bench/src/lib/usage.rs

```diff
@@ -82,6 +82,25 @@ impl BenchmarkUsage {
 			_ => None,
 		}
 	}
+
+	pub fn to_json(&self) -> color_eyre::eyre::Result<String> {
```

Comment:
This is more like, to_chart_items, because you also have Serialize/Deserialize derived for this structure which ca produce Jsons